labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,50 @@
+function startGame() {
+  const play = window.prompt("Do you want to play?");
+  if (play === "yes") {
+    var name = window.prompt("Enter your name:")
+    startCombat(name);
+  }
+}
+
+function getDamage() {
+  const min = 1;
+  const max = 5;
+  return (Math.floor(Math.random() * (max - min + 1) + min));
+}
+
+function startCombat(name) {
+  var winCount = 0;
+  var userHP = 40;
+  var grantHP = 10;
+  var choice;
+
+  while ((winCount < 3) && (userHP > 0)) {
```

Comment:
What if you broke out a function that handled one round of combat, and kept it inside startCombat() so it has access to userHP and grantHP?